lib/Conversion/StableHLOToTTIR/StableHLOToTTIRPatterns.cpp

```diff
@@ -1057,6 +1058,445 @@ class StableHLOToTTIROpLogicalOpConversionPattern
   }
 };
 
+template <typename srcOpTy>
```

Comment:
nit: first letter should be upper case for template names

Reply:
Thanks. I will fix it.